### DISCUSSION TITLE
Welcome to fabric-gm-wiki Discussions!

### DISCUSSION THREAD

**Discussion by davidkhala** [Hyperledger-TWGC/fabric-gm-wiki#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SamYuan1990** [comment]
提议：
1. netty java 库用pku c库编译
1. 互操作库添加 ccs库为基础的校验
1. 互操作库添加文件级别互操作校验
1. 互操作库添加sm3/4 校验

**Reply by davidkhala** [reply]
point 2, 3, 4 have no uncertainty for discussion..

**Reply by SamYuan1990** [reply]
to be honest, just have a try to git hub discussion feature.
建议我们用中文书写，并且在周会上提议一下吧。

**Reply by SamYuan1990** [reply]
另外，我不确定我们要用这个来讨论什么？ 
相较于我直接开个issue来描述某个功能？或者某个实现/问题。

**Reply by davidkhala** [reply]
这个其实是我之前实验性质地点开来的，可以关掉了
